examples/01_plotting/plot_demo_plotting.py

```diff
@@ -45,12 +45,14 @@
 
 from nilearn import plotting
 
+# %%
```

Comment:
doing some changes to make sure that the examples that crashed on main can be run now